core/src/main/kotlin/model/Documentable.kt

```diff
@@ -57,8 +62,13 @@ class Property(
     override val receiver: Parameter?,
     override val expected: PlatformInfo?,
     override val actual: List<PlatformInfo>,
-    override val extra: MutableSet<Extra> = mutableSetOf()
-) : CallableNode() {
+    override val extra: MutableSet<Extra> = mutableSetOf(),
+    val type: TypeWrapper?,
+    val accessors: List<Function>,
+    val isVar: Boolean = false,
```

Comment:
What if var is different on different platforms? `open class A`  and `open val x : String` in common and actual `var x: String`

Reply:
Removed this information.